### ISSUE TITLE
Remove NFS diagnostic check?

### ISSUE THREAD

**Issue by tsteur** [matomo-org/matomo#15259]
It says eg 

> Warning Filesystem: warning (Your server is using an NFS filesystem. This means Matomo will be extremely slow when using file based sessions.)

But we are not using file based sessions anymore anyway. 

Could also simply remove the message but not sure what the point of having this check is then. Of course could say in general using NFS may be slow.

**Comment by mattab** [comment]
Reckon the diagnostic is still valuable, even when not using file sessions, because even just storing the tmp/ cache folder in NFS results in slow downs. Maybe we could change the second sentence to ` This means Matomo will load slower than it should be.`

**Comment by hoermannklaus** [comment]
I am using Matomo in version 4.7.1 and still get the Warning "Your server is using an NFS filesystem. This means Matomo will be extremely slow when using file based sessions."
I though as defacto standard matomo uses database bases sessions.
Why am I seeing this warning then?

I want to have a nice system check, with all green checkmarks. :-)

**Comment by sgiehl** [comment]
Hi @hoermannklaus. You are right, the message is still a bit missleading. But as mentioned above the Matomo UI might still work a bit slower than normal on NFS, as all templates and assets are stored in the `tmp` path

**Comment by hoermannklaus** [comment]
Yes I read the above statement. But this is nothing I could change, as I deploy Matomo on our internal cluster. And I am not in the position to make changes to the underlying storage (and I do not want to).
So I guess I have to live with this fact, not having a "clean" system check.
